example/feedback/feedback.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Bucket feedback collection</title>
+    <script src="https://cdn.jsdelivr.net/npm/@bucketco/tracking-sdk@1"></script>
+    <style>
+      fieldset label {
+        display: flex;
+        align-items: center;
+        padding: 5px;
+        gap: 5px;
+      }
+
+      label input {
+        margin: 0;
+      }
+    </style>
+  </head>
+  <body>
+    <script>
+      function handleSubmit(e) {
+        e.preventDefault();
+
+        const formData = Object.fromEntries(new FormData(e.target).entries());
+
+        const feedbackPayload = {
+          featureId: "EXAMPLE_FEATURE",
+          userId: "EXAMPLE_USER",
+          companyId: "EXAMPLE_COMPANY",
+          score: formData.score ? Number(formData.score) : null,
+          comment: formData.comment ? formData.comment : null,
+        };
+
+        // Using the Bucket SDK
+        bucket.feedback(feedbackPayload);
+
+        // Using the Bucket API
+        fetch("https://tracking.bucket.co/EXAMPLE_TRACKING_KEY/feedback", {
+          method: "POST",
+          headers: {
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify(feedbackPayload),
+        });
+      }
+    </script>
+
+    <form action="#" onsubmit="handleSubmit(event)">
```

Comment:
do you need `(event)` here?

Reply:
Yeah. Old style inline event handling. Saves us from the additional indirection of having to attach an event handler with a querySelector in the JS. I'd like to keep the example as focused on the relevant parts as possible